app/views/courses/_basic_details_tab.html.erb

```diff
@@ -245,7 +249,7 @@
             Allocations
           </dt>
           <dd class="govuk-summary-list__value govuk-!-padding-right-0" data-qa="course__allocations_info">
-            <% if course.subjects.include?('Physical education') %>
+            <% if course.subjects.map(&:subject_name).include?('Physical education') %>
```

Comment:
can we shove it in

app/decorators/course_decorator.rb

it already does sorted_subjects...